### PR TITLE
Conversion of widget code editor tests from cypress to selenium

### PR DESCRIPTION
**Pull request by radhikav1**
# [CDAP-19253](https://cdap.atlassian.net/browse/CDAP-19253) : Convert widget.codeeditor.spec.ts test from cypress to selenium

## Description
Conversion of widget code editor tests from cypress to selenium.

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [x] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-19253](https://cdap.atlassian.net/browse/CDAP-19253)

## Test Plan
Included in cdap UI e2e test

## Screenshots
n/a

### CODE REVIEW

File: src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java [hunk 1]
```diff
@@ -67,6 +67,23 @@ public static void connectTwoNodes(NodeInfo source, NodeInfo target) {
...
 
+  public static void openNodeProperty(NodeInfo node) {
+    String elementId = "plugin-node-" + node.getNodeName() + "-"
+            + node.getNodeType() + "-" + node.getNodeId();
+    WebElement nodeElement = Helper.locateElementByTestId(elementId);
+    WebElement versionElement = nodeElement.findElement
+            (By.cssSelector("div[class*='node'] > div[class*='node-metadata'] > div[class*='node-version']"));
+
+    JavascriptExecutor jsExecutor = (JavascriptExecutor) SeleniumDriver.getDriver();
+    jsExecutor.executeScript("arguments[0].style.visibility='hidden'", versionElement);
```
Comment (sumengwang) [inline src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java]: what are these two scripts doing?

Comment (mrahanjam) [reply src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java]: You can use `CdfStudioActions.navigateToPluginPropertiesPage(node.getNodeName());` to open a node property page by its name;

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/WidgetCodeEditor.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    public void openJSNodeProperties() {
+        Commands.openNodeProperty(jSNode);
+    }
+    @Then("Get JS editor value and compare with default JS editor value")
+    public void verifyDefaultJSEditorValue() {
+        WebElement jsEditorElement = Helper.locateElementByCssSelector("div[class*='ace-editor-ref']");
+        WebElement jsEditorContentElement = jsEditorElement.findElement(
+                By.cssSelector("div[class*='ace_text-layer']"));
+        String editorValue = ElementHelper.getElementText(jsEditorContentElement);
+        Assert.assertEquals(StringUtils.normalizeSpace(editorValue),
```
Comment (mrahanjam) [inline src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/WidgetCodeEditor.java:76]: is the `StringUtils.normalizeSpace` needed? doesn't `editorValue.equals(defaultJsEditorVal)` work?

**Review by mrahanjam — APPROVED** [review]
Left a couple of comments, but lgtm overall

File: src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/WidgetCodeEditor.java (hunk 2 above)
Comment (radhikav1) [reply src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/WidgetCodeEditor.java:76]: yes, equals(..) is not working here, so had to go for `StringUtils.normalizeSpace` .

File: src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java (hunk 1 above)
Comment (radhikav1) [reply src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java]: Those two scripts are not needed anymore as method is replaced with `CdfStudioActions.navigateToPluginPropertiesPage` now as suggested.

**Review by sumengwang — APPROVED** [review]

**Pull request merged**